conformance/setup.go

```diff
@@ -510,8 +512,10 @@ func init() {
 	automaticCrossmountVal, runAutomaticCrossmountTest = os.LookupEnv(envVarAutomaticCrossmount)
 	automaticCrossmountEnabled, _ = strconv.ParseBool(automaticCrossmountVal)
 
-	reportJUnitFilename = "junit.xml"
-	reportHTMLFilename = "report.html"
+	if dir := os.Getenv(envVarReportDir); dir != "none" {
```

Comment:
What happens when `dir == "none"`, and when it's not set? Do we still want the original defaults?

Reply:
I thought it best not to change the existing behaviour because people might well be relying on it in scripts or CI, so it remains the same: when $OCI_REPORT_DIR is empty or not set, the file names are just `junit.xml` and `report.html` respectively, because `filepath.Join("", x)` is `x`.

Reply:
Does that mean `OCI_REPORT_DIR=none` is a way to disable reporting?

Reply:
Yes, that's correct.